packages/web/src/components/spa/HackAUBG/MentorsSection/MentorsSection.jsx

```diff
@@ -26,11 +24,10 @@ export const MentorsSection = () => {
         if (mentor) {
             return (
                 <div className="mentors-section-container">
+                    <img className="pacman-left" src="Pacman-left.png"></img>
```

Comment:
you can try wrapping the img and h1 tag in a div and making it display flex to align the img with the title